atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -73,9 +73,9 @@ export const fatalityGridTableColumns = {
   engineering_area: {
     searchable: true,
     sortable: true,
-    label_table: "Engineering Area",
+    label_table: "Current Engineering Area",
```

Comment:
based on Xaviers comments in release party today

Reply:
I wish i recalled exactly what the context was around this, because adding the "current" makes it feel like sometimes we operate on "previous" or "future" Engineering Areas, which doesn't make a ton of sense to me. But it's totally all good - not worth digging into.

Reply:
I thinkk Xavier wanted to capture the fact that the engineering areas could change at some point and that the VZE is operating off of the "current" areas rather than past ones?